assets/mail-templates/failure.txt

```diff
@@ -0,0 +1,17 @@
+✗ Echidna failed to publish @SOURCE
+  for the following reasons:
+
+@ERRORS
+More details are available here@ATTACHMENT:
+
+  https://labs.w3.org/echidna/api/status?id=@ID
```

Comment:
I wonder if we should use `global.BASE_URI` or `process.env.BASE_URI` here, instead of hard-coding the URL?

Reply:
I think that's ok to keep the URL in the template unless we think it'll be used in many different places.